tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cuda12.yml

```diff
@@ -8,7 +8,7 @@ stages:
     torch_version: '2.1.0'
     opset_version: '17'
     cuda_version: '12.2'
-    cmake_cuda_architectures: 70;75;80;86;90
+    cmake_cuda_architectures: 80;86;90
```

Comment:
Is this the minimum number of architectures we can remove to get the wheel size under 300 MB? I'm a bit concerned about getting rid of 75 since it looks like it's still the default, but I'll defer to whatever Baiju thinks is best: https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_ARCHITECTURES.html.

Reply:
outdated